components/bookingForm/bookingForm.tsx

```diff
@@ -94,18 +96,24 @@ export const BookingForm = ({ recaptchaKey }) => {
       contactReCaptcha,
       sourceWebPageURL
     );
+
+    const method = { Method: "Create-Lead-UI" };
     actions.setSubmitting(false);
 
     await axios
       .post("/api/create-lead", data)
       .then((response) => {
         if (response.data && !response.data.success) {
+          response.data = { ...response.data, data };
+          appInsights?.trackException(response.data, method);
```

Comment:
```suggestion
          appInsights?.trackException({ input: response.data, response: data }, method);
```
the original line is difficult to read, does my suggestion convey the same intent?

Reply:
It only accepts IExceptionTelemetry or Axios response. So your suggestion above would not work.

Reply:
I think the multiple reference to "data" is throwing me off. Can we then rename the variable or make it more clear via a type?